_docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -22,6 +22,8 @@ This Quick Start creates a new GKE cluster, installs Istio and then deploys the
 This sample requires a valid Google Cloud Platform project with billing enabled.  
 - If you are not an existing GCP user, you may be able to enroll for a $300 US [Free Trial](https://cloud.google.com/free/){:target="_blank"} credit.
 
+- Ensure that the [Google Container Engine API](https://console.cloud.google.com/apis/library/container.googleapis.com/) is enabled for your project. You can check to see if the API is enabled by viewing the API in the API Library (found by navigating to "APIs &amp; Services" -> "Dashboard" in the navigation bar).
```

Comment:
in jekyll launches a new window:

[Google Container Engine API](https://console.cloud.google.com/apis/library/container.googleapis.com/){:target="_blank"}

the link here automatically takes you to the 'enable api' section so maybe just users can enable and check with that link alone on the project they just created or are using

Reply:
Added {:target="_blank"}

Also shuffled the wording up a bit so hopefully it makes more sense now.